docs/data-access/icgc-25k-data.md

```diff
@@ -22,15 +22,18 @@ Files from other contributing projects are all hosted by ICGC's partner reposito
 
 ## Accessing ICGC 25K Release Data
 
-A SFTP server is available to access ICGC Release Data and PCAWG data.
+ICGC Release Data and PCAWG data are hosted in two locations: an SFTP server, and an AWS bucket.
 
-The server hosts three data directories with the following data:
+The SFTP contains public and DACO controlled data, only available to authorized DACO-approved users only. If you previously had DACO access for ICGC 25K data you will continue to have permission to access the SFTP server. If you require DACO approval please see the documentation on [applying for DACO access](./daco/applying.md).
+
+The AWS bucket contains public data and can be accessed without DACO approval.
+
+Both locations contain directories with the following data:
```

Comment:
Let's rearrange this text to state upfront that there are different sources for open and controlled data:

```md
ICGC Release Data contains both open and controlled access data. [Open access release data](#link-to-open-access-header) is stored on a publicly available Object Storage Bucket. [Controlled access release data](#link-to-controlled-access-header) is hosted on an SFTP server that requires ICGC DACO approval to access.
```